.github/workflows/release-python-client.yaml

```
@@ -0,0 +1,55 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: set semver
+        run: |
+          CURRENT_SEMVER=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
+          echo "CURRENT_SEMVER=$(echo ${CURRENT_SEMVER})" >> $GITHUB_ENV
```

I don't totally know if this is a reliable way to write a variable to be used in a later step. I *think* the more generally recommended way is to add a [variable to the output](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter). 

```suggestion
      - name: set semver
        id: "set_semver"
        run: |
          CURRENT_SEMVER=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
          echo "::set-output name=CURRENT_SEMVER::$CURRENT_SEMVER"
```

---

If you can use the suggestion, then you won't need this step! And the reference below will look a little different - something like `${{ inputs.current_semver }}`

---

Github has changed the syntax around variables a few times. You can use @snf2ye 's suggestion or simplify the export of the environment variable export as follows: `echo "CURRENT_SEMVER=${CURRENT_SEMVER}" >> "$GITHUB_ENV"` . One nice thing about using environment variables is you can see which environment variables exist at each step, and they're immediately available in every subsequent step.